tests/apollo/util/eliot_logging.py

```diff
@@ -0,0 +1,32 @@
+from eliot import start_action, start_task, to_file, add_destinations, log_call
+from datetime import datetime
+import os
+import sys
+
+
+def set_file_destination():
+    storage_type = os.environ.get('STORAGE_TYPE')
+    test_name = next(f"{m}_{storage_type}" for m in sys.modules.keys() if "test_" in m)
```

Comment:
In some cases (when Apollo is running as part of Hermes), we load multiple Apollo test modules. Could you also incorporate this case here? 

Essentially, if we have more than one module called "test_", we can set `test_name` to a generic file name (say "apollo_run_" + a timestamp).